Alloy/commands/compile/index.js

```diff
@@ -174,17 +174,18 @@ module.exports = function(args, program) {
 		path.join(paths.resources, titaniumFolder, "alloy", "backbone.js")
 	);
 
-	// Generate alloy.js from template
-	var libAlloyJsDest = path.join(paths.resources, titaniumFolder, 'alloy.js');
-	var pkginfo = require('pkginfo')(module, 'version');
-	logger.trace('Generating ' + path.relative(titaniumFolder, libAlloyJsDest).yellow);
-	fs.writeFileSync(
-		libAlloyJsDest,
-		ejs.render(
-			fs.readFileSync(path.join(alloyRoot, 'template', 'lib', 'alloy.js'), 'utf8'),
-			{ version: module.exports.version }
-		)
-	);
+	if (restrictionPath === null || restrictionPath === path.join(paths.app, 'alloy.js')) {
+		// Generate alloy.js from template
+		var libAlloyJsDest = path.join(paths.resources, titaniumFolder, 'alloy.js');
+		logger.trace('Generating ' + path.relative(titaniumFolder, libAlloyJsDest).yellow);
+		fs.writeFileSync(
+			libAlloyJsDest,
+			ejs.render(
+				fs.readFileSync(path.join(alloyRoot, 'template', 'lib', 'alloy.js'), 'utf8'),
+				{ version: module.exports.version }
```

Comment:
@xavierlacot, thank you for catching this! 
Perhaps we don't want to pass an empty version number when generating the file.
This is missing: 
```var pkginfo = require('pkginfo')(module, 'version');```